src/NCDialog.cc

```diff
@@ -1095,6 +1096,24 @@ void NCDialog::processInput( int timeout_millisec )
 
 		break;
 
+	    case KEY_F( 18 ): // = Shift-F6 on e.g. a linux console
+                {
+                    yuiMilestone() << "Asking for widget ID" << std::endl;
+                    YWidget * widget = YNCursesUI::ui()->askSendWidgetID();
+
+                    if ( widget )
+                    {
+                        NCPushButton * button = dynamic_cast<NCPushButton *>( widget );
```

Comment:
so only buttons can be selected? not radio buttons and others? I think we should at least log if widget with given ID exists, just it is not button, so we do nothing.

Reply:
The first thing that happens is that the widget gets the keyboard focus. This is done for all widgets. But if it's a button, it is also immediately activated just as if the user pressed [Return].